science/2022-11-23-science-blog-6/index.md

```diff
@@ -0,0 +1,15 @@
+---
+page-title: "What else does a red warning say when incidence is high?"
```

Comment:
@MikeMcC399 Is "else" here correct? In German the title is meant to question the usefulness of a red tile in times of high incidences. For this "noch" is used in the German title, which was translated to "else". That doesn't seem right to me.

Reply:
I suggest instead "What does a red warning mean when the incidence is high?"